apps/ledger-live-mobile/src/components/DeviceAction/index.tsx

```diff
@@ -4,8 +4,27 @@ import type { Action, Device } from "@ledgerhq/live-common/hw/actions/types";
 import { DeviceNotOnboarded } from "@ledgerhq/live-common/errors";
 import { TransportStatusError } from "@ledgerhq/errors";
 import { useTranslation } from "react-i18next";
-import { useNavigation, useTheme } from "@react-navigation/native";
+import {
+  ParamListBase,
+  useNavigation,
+  useTheme,
+} from "@react-navigation/native";
 import { Flex, Text } from "@ledgerhq/native-ui";
+import type { AppRequest } from "@ledgerhq/live-common/hw/actions/app";
+import type { InitSellResult } from "@ledgerhq/live-common/exchange/sell/types";
+import { TokenCurrency } from "@ledgerhq/types-cryptoassets";
+import type { AccountLike, DeviceInfo } from "@ledgerhq/types-live";
+import { Transaction } from "@ledgerhq/live-common/generated/types";
+import {
+  Exchange,
+  ExchangeRate,
+  InitSwapResult,
+} from "@ledgerhq/live-common/exchange/swap/types";
+import { AppAndVersion } from "@ledgerhq/live-common/hw/connectApp";
+import { LedgerErrorConstructor } from "@ledgerhq/errors/lib/helpers";
```

Comment:
PTSD triggered: is the `lib` in this import expected ?

Reply:
Yes, for `@ledgerhq/errors` it is fine.

As a matter of fact the rule is: check the package.json file and if there is an `exports` field then it is not ok to put `/lib` in the path.

Reply:
Ah yeah, i checked live-common and it's [here](https://github.com/LedgerHQ/ledger-live/blob/c4b16f3cf304e726692cda66858ef4cd5967f671/libs/ledger-live-common/package.json#L41)
Thanks a lot !